tests/integration/api_notification_test.go

```diff
@@ -154,7 +154,7 @@ func TestAPINotification(t *testing.T) {
 		AddTokenAuth(token)
 	resp = MakeRequest(t, req, http.StatusOK)
 	DecodeJSON(t, resp, &newStruct)
-	assert.True(t, newStruct.New == 0)
+	assert.Equal(t, 0, newStruct.New)
```

Comment:
```suggestion
	assert.EqualValues(t, 0, newStruct.New)
```


`newStruct.New` seems to be a `int64`

Reply:
(or `assert.Zero`)

```suggestion
	assert.Zero(t, newStruct.New)
```